CONTRIBUTING.md

```
@@ -0,0 +1,455 @@
+
+The triage team are active OpenAPI members who help with discussion and issue management.
+They respond to new issues and discussions, direct people to our existing resources or raise conversations to a wider audience.
+The triage team keeps an eye on the backlog and closes issues and discussions that are no longer active or needed.
+
+## Get in touch
+
+To get in touch with other people on the project, ask questions, or anything else:
+
+- Find us [on the OpenAPI Slack](https://communityinviter.com/apps/open-api/openapi).
+- Start a [GitHub Discussion](https://github.com/OAI/OpenAPI-Specification/discussions/).
+- Join one of our weekly meetings by checking the [issues list for an upcoming meetings](https://github.com/OAI/OpenAPI-Specification/issues?q=is%3Aissue%20state%3Aopen%20label%3AHousekeeping).
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)